README.md

```diff
@@ -60,3 +76,15 @@ Steps:
 1. Click the ``Open in Gitpod`` button.
 2. Relax: a development environment with an active Wagtail Guide site will be created for you.
 3. Login at `/admin/` with username `admin` and password `changeme`
+
+
+# Contributing
+
+If you're a python or Django developer, fork the repo and join us. You'll find answers to many common new contributor questions in our [contributing guidelines](https://github.com/wagtail/bakerydemo/blob/master/contributing.md).
```

Comment:
Not sure, but should Python be capitalised because it is a product name? We also write Django and Wagtail.